.github/workflows/ci.yml

```diff
@@ -57,15 +57,15 @@ jobs:
     - name: Get libc version
       run: ldd --versio
 
-    - name: Get branch name for pull request
-      if: ${{ github.event_name == 'pull_request' }}
+    - name: Get branch name to match with other repos
       run:  |
-          echo "GIT_BRANCH=$GITHUB_BASE_REF" >> $GITHUB_ENV
-
-    - name: Get branch name for push
-      if: ${{ github.event_name == 'push' }}
+          echo "GIT_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
```

Comment:
This did not have the intended effect.

![image](https://github.com/user-attachments/assets/3474706d-9139-49c6-a5bd-1b8cef29b235)

SSC won't have an 1827/merge ref, so this will always default to patch. I think this does return the correct value for a push, but not a pull_request.

Is the right move to override with `github.head_ref` in the if block below?